semgrep-core/src/parsing/tree_sitter/Parse_bash_tree_sitter.ml

```diff
@@ -1177,7 +1177,7 @@ let parse file =
   H.wrap_parser
     (fun () ->
       Parallel.backtrace_when_exn := false;
```

Comment:
Oh can we also kill these then?

Reply:
You mean the setting of Parallel.backtrace_when_exn? Yes!